### ISSUE TITLE
german translations author typxxi

### ISSUE THREAD

**Issue by dckiller51** [dckiller51/lovelace-body-miscale-card#2]
EVALUATION OF LATE NIGHT BUILT 0620 
=============================
First note goes to the built number: 0620-hh should be the new sheme where hh are representing the hour  of release cause I had yesterday only gotten 0620s

I will start with pure translation points except that I wanted to start from top to bottom and therefore begin with a non translation issue but a more general one which you had already answered and I added your reply as state: not possible yet.

I. TRANSLATIONS
=============================

**#1**
card configuration title should be 
`"Lovelace body miscale Card Configuration"`

For me it should be the spot #1 for the BUILT number like cause I was struggling a lot what code my card was relying on which is very frustrating to not be able to get behind which code is running except by differentiating that due to its translation errors  
`"Lovelace body miscale [0620-hh] Card Configuration"`  
STATE: post poned cause right now not possible

**#2**
A new error or untranslated term had appeared behind the scale icon "unknown" 
![image](https://user-images.githubusercontent.com/20662430/122768868-4f30cd80-d2a4-11eb-9656-140cc5470553.png)

I guess we need to call that "STATUS: unbekannt"  (means "STATE: unknown")
Therefore the previous "OK" value has to change too into "MESSUNG: OK" (means "MEASUREMENT: OK"). 
That way both messages are accurate for its purpose, first for scale in general, second is about the measurement results.

Search for 
`"OK"`
and replace the whole german term by this
`"MESSUNG: OK"`

And regarding "unknown" you have to implement that on your own as "State: unknown" - for german "STATUS: unbekannt"
Do not forget in case of errors to show behind the scale icon the state message "State: error" - for german "STATUS: Fehler", at least as long as both icons are shown at the same time. 

**#3**
Search for 
`"Auswahl der Waage / Person erforderlich"`
and replace the whole german term by this
`"Bitte ein Waagen Konto auswählen [erforderlich] !"`

After being unsatisfied with the previoius term (which is right but feels weird to read and explain) we now could go the technical way which would mean a "Sensor - user name" combination or much easier "Please choose your scale account [mandatory] !"

You have a scale and even if you are alone the system it can only work if you have created at least 1 account in that bodymiscale.yaml file - can you tell me that a second user has to be added in that file too ?

**#4** 
Search for 
`"Bild (Optional)"`
and replace the whole german term by this
`"Hintergrundbild [optional]"`

For me it is not about "pic" but about a "background pic" cause the figures will lay in front that (background image). 

**#5** 
Search for 
`"Detaillierte Messergebnisse anbieten" `
and replace the whole german term by this
`"Detaillierte Messergebnisse anbieten (untere Hälfte - per Power Button einblenden)"` 

This means "Offer detailed measurement results (lower part - press power button to show)"

As said before it would be nice if there is a way to add a carriage return line feed / new line after "... anbieten"
`"Detaillierte Messergebnisse anbieten`
`(untere Hälfte - per Power Button einblenden)"`
I had seen that before but I can not tell  you where.

**#6**
Search for 
`"Schaltfläche anzeigen" `
and replace the whole german term by this
`"Schalter (U) anzeigen (rechts unten)"`

Previous "Schaltfläche" means a (bigger) switch area like 'cancel' or so, but what you mean is that U spot / button which is called in german "Schalter" / button and where I added (U) cause it looks like an "U" in a white circle switch.

**#7**
Search for 
`"Symbolleiste"`
and replace the whole german term by this
`"Symbolleiste anzeigen (Power Button und Schalter)"`

I had to change the terms inside to "Power Button" (the translation "Einschaltknopf" sounds weird in that context cause it would mean to turn the whole scale on / off which it does not do). 

Therefore I still suggest to use any different kind of button instead that particular turn on / off button which will cause questions.

I still recommand an icon that represents to hide / unfold something like a triangle or twister ᐅ to press for more details and if that had been pressed I suggest to replace that previous ᐅ with the opposite twister / triangle like this ▼ cause this switch has a toggling meaning to show or hide details. 
A power button on the other side would always target on a device as a whole like turn on / off your complete HiFi not a particular feature on that stereo. 

**#8**
Search for 
`"Bitte beachten ...."`
and replace the whole german term by this
`"ACHTUNG: Weitere Optionen sind nur im Code Editor verfügbar. [0620-hh]"`

which means
`"ATTENTION: Further options are only available in the code editor. [0620-hh]"`

QUESTION: I have deleted the word "actions" cause I was not sure if there are any actions available or not and if actions have to be mentinoned there cause usually it will cause questions like "What actions ?" 

I'd prefer the short and simple one cause that would work better on tablet or smartphones too and would not lead to ask what other "actions" are available. 

**UPDATE: 21st 17:00** 
**#9#**
This translation in the beginning of the former translation file has been lost cause the translation had been already there and is now replaced by the oldest  or first one. 
"name": "BodyMiScale Karte",
"description": "Die BodyMiScale Karte kann Ihnen Ihren gewichtsmäßigen Körperstatus anzeigen."

II. GENERAL CONFIGURATION
=============================

This chapter is beyond translations, more integral questions in general like in which order why option should appear or even been hidden cause it makes no sense to offer switches that will not be shown if the general symbol bar is off, but that explanation will follow below.

Just an example that you first need a symbol bar that is turned on to then make those other switches available.

**I.**
You need a new first line as explanation of the general behaviour
BIG LETTERS: CHANGES WILL ONLY APPEAR AFTER BEING SAVED.

Why is that important ? 
I had struggled with it but only in certain cases. While in configuration you can press any of those 6 switches but nothing will change before you will have saved it. But on the right half iof the configuration dialog you have a preview and for example that power button in the preview will be working regardless how the switches are switched or not.  

**II.**
Please move that "model number" input line above that "Background image" line for the simple reason that all mandatory inputs should get the highest amount of attention in the top section and should be selected first followed by the less important optional ones like that Background image.

**III.**
I would love to see a better structure in the configuration especially regarding these in total 6 switches.

![image](https://user-images.githubusercontent.com/20662430/122775510-7ee2d400-d2aa-11eb-8f16-0cc8f7dc1ef1.png)

Therefore can you please add an empty line (or separator line out of ----) above the first switch which is followed by this line
```"Options for card header" = "Kartenkopf Optionen"```

**IV.** 
After these first 3 switches I would love to see 
an empty line
followed by this additional  
`"Further card options (measurement details / switches)" or in german "Weitere Kartenoptionen (Messung Details / Schalter)"`
Yes, both german versions are this time shorter !

It will make it a lot easier in the beginning to understand where the targets of the upper half are and what each switch is for while we both do it know now blindly due to daily training but the average user will struggle which swtich is what for (therefore I had pointed out where those things are located).

**V.** 
Right now the switches 4, 5, 6 are not in the RIGHT and an INTUITIVE order !
`4 show button for more measurement details `
`5 show switch`
`6 show symbol bar`

But that LAST SWITCH "show symbol bar" has to be the first choice in the lower section cause **without showing the symbol bar** switches 4 and 5 are useless to click on and therefore should stay hidden / unavailable, right ?

Only once "symbol bar" is activated the other 2 switches should appear cause only then they can work.
Without icon bar enabled the "show button" and "show switch" are running into the emptineess as we called it here.
`4 Show symbol bar`
`5 Offer button for more measurement details` 
`6 Show switch`

I chose "offer button for more measurement details" cause you only offer a button which has to be pressed to show further measurement details. This is not a general switch to always show the measurement details. "show" instead "offer" will be misunderstood cause if you would activate that switch "show button for more measurement details" people will mostly only read and expect "show more measurement details"

**VI.**
The **"Switch (U)"** should get a title like **"Show scale account selector(s)"** and that would finally mean to replace "show switch" by "Show scale account selector(s)" ? 
ATTENTION: if you already have done the translation tasks mentioned above in the first section then you will have to do this

Search for 
`"Schalter (U) anzeigen (rechts unten)"` 
and replace the whole german term by this
`"Den / Die Waagekonten Auswahl Schalter (U) anzeigen (rechts unten)"`

**VII.**
Regarding the **model number input** I was not happy too and I am not convinced about that way you had implemented cause in reality it is - at least right now - just a choice between 2 scales and nothing more. Therefore: 

**Isn't a switch a better solution to chose between 2 models where '181D' should be the default setting?**
I suggest as title "**Do you own model 181B incl. impedance measurement?**"

By default it is turned of and therefore it will run as a model 181D user. 
Everyone with a better model does not loose much with such basic setting and can switch easily to the better 181B to get more measurement details.
If the "poor" 181D owner turns on this switch for the 181B model he will get a lot more label but no values and learns what the differences are between those similar looking scales - as I had learned it too from you and your code / data. 

That typing solution is not ideal and it would never be my first choice even though you read the initial value out of the yaml file and you might write that value back too. Such switch would work at least right now a lot better and looks nicer too for sure.
 
**AND:** a **switch does not need the additional "mandatory" term** too cause with a switch the card would start by default as 181D.

**VIII.**
I guess we have to change the order of the detailed measurement results in the lower half.

BMI should be followed by 
BMI classification
FINALLY: Age can be deleted from that measurement detail page cause it is not really needed there.  

III. POINTS TO NOT FORGET
=============================

**1. VARIABLE OR LABEL NAME with TYPO**
I saw in the translation file this "Lack-exerscise": "Bewegungsmangel" and exercise is spelled without the s in "rsc"

**2. ALL ABOUT FIGURES AND FORMATS**
* usefull, consistent decimals like 2 decimals for weight
* metabolism kcal figure with national / domestic thousand separators like german 1.856 kcal instead of 1,856 kcal
* central european figures with a "," as decimal separator cause 102.3 is not right here
* long term: user CONFIGUREABLE DECIMALS in bodymiscale.yaml

**3. Aligned lines for Status (upper left)**
For the left upper status we should make sure that the users gets just 1 icon (scale or warning, but never both cause they are contradicting) in case things are going wrong like for example in a measurement.
181B owners can get longer and misaligned error messages that go over 2 lines where the 2nd line starts
even below that warning icon instead of being aligned to the first line.
![image](https://user-images.githubusercontent.com/20662430/122782583-219e5100-d2b1-11eb-87ce-00f17f2ae166.png)

**Comment by typxxi** [comment]
I can not edit that moved topic anymore so you might have to give me access cause everything has been put in that topic. I just wanted to add the point that the whole decimal separator questions have to be solved inside the custom lovelace cards itself, but I can not without the rights. 
If it is not possible to give me the editor rights I might copy it from the old topic and paste it here and then you have to shorten the first one to a shorter introduction. I simply want to have 1 spot where all is put together and tracked between these 3 major categories.

**Comment by dckiller51** [comment]
#2 add "MEASUREMENT: OK" and STATE: unknown"

#3 add "Please select a scale account (required)"
I like your modification better than the old one. I kept the spirit of the other cards with the () and without the !

#4 add "Background image (optional)"

#5 I have not made any changes at the moment. I want to add a mdi:chevron-down and mdi:chevron-up

#6 I have not made any changes at the moment. The idea is that one icon = 1 user. You can easily replace this icon by any letter. mdi:alpha-a-circle for Aurélien or mdi:alpha-s-circle for Siham... My goal is to be able to put several bodymiscale.name so there is a possibility to activate 5 buttons. You can try by looking at the above
```
type: custom:body-miscale-dev
entity: bodymiscale.aurelien
model: 181B
show_name: true
show_state: true
show_body: true
show_buttons: true
show_toolbar: true
show_attributes: true
image: /local/body-miscale-card/miscale2.jpg
buttons:
  user1:
    label: Aurélien
    icon: mdi:alpha-a-circle
  user2:
    show: true
    label: Siham
    icon: mdi:alpha-s-circle
```
#7 See #5

#8 add "ATTENTION: Additional options are only available in the code editor. [0622-01:00]"
This corresponds to example #6 where we are forced to draw manually to have more options in the card

#9# made

**Comment by dckiller51** [comment]
@typxxi 
Now you should have the rights. I needed you to post
A first release is available on the github dev

**Comment by typxxi** [comment]
First of all: I guess that if you wrote "add" you meant to say that you have added that point , right ?


> #3 add "Please select a scale account (required)"
> I like your modification better than the old one. I kept the spirit of the other cards with the () and without the !
If things are important they deserve an "impetus" or ! as long as there is no inflation and a requirement deserves a '!' cause then no user has a reason to complain.

> #5 I have not made any changes at the moment. I want to add a mdi:chevron-down and mdi:chevron-up
Well, I know what you meant to say but it is better to get the translations done early to see how they look and 5 is mostly just about a better translation with 

_Search for
"Detaillierte Messergebnisse anbieten"
and replace the whole german term by this
"Detaillierte Messergebnisse anbieten (untere Hälfte - per Power Button einblenden)"_

If the power button will become something new like a twister or so it is easy to exchange that single word. 
And regarding that CRLF part for 2 lines it is just a point that also can be achieved later. For me it is about getting the whole picture and familiar with the translation how that fits into it.

_As said before it would be nice if there is a way to add a carriage return line feed / new line after "... anbieten"
"Detaillierte Messergebnisse anbieten
(untere Hälfte - per Power Button einblenden)"_

> #6 I have not made any changes at the moment. The idea is that one icon = 1 user. You can easily replace this icon by any letter. mdi:alpha-a-circle for Aurélien or mdi:alpha-s-circle for Siham... My goal is to be able to put several bodymiscale.name so there is a possibility to activate 5 buttons. You can try by looking at the above
Ok, that was what I was waiting / hoping for cause then I have to prepare my translations for all such cases like 1 or multiple user switches even though it makes only sense for at least 2 cause 1 only can not switch to another. So I would suggest that you will only offer 2 - 5 not just 1 alone cause only 2 can explain the purpose without any further explanations.

The following lines are those I would have to add for a multiple user scale card, right ?
> type: custom:body-miscale-dev
> entity: bodymiscale.aurelien
> model: 181B
> show_name: true
> show_state: true
> show_body: true
> show_buttons: true
> show_toolbar: true
> show_attributes: true
> image: /local/body-miscale-card/miscale2.jpg
> buttons:
>   user1:
>     label: Aurélien
>     icon: mdi:alpha-a-circle
>   user2:
>     show: true
>     label: Siham
>     icon: mdi:alpha-s-circle
according to the rule of thumb above it would mean that you will not need "show: true" cause the first 2 have to be visible always in case you activate those user selectors.

> #7 See #5
I suggest to simply add or better replace these lines and we can later adopt the term depending on which switch you had found to be the most usefull. Power Button is used in german too and therefore no issue. If you have found the final icon we can change that description. I simply want to keep that translation list as short as we can cause it is a space eater, lot of lines.

> #8 add "ATTENTION: Additional options are only available in the code editor. [0622-01:00]"
> This corresponds to example #6 where we are forced to draw manually to have more options in the card
Same as above: For me it is about getting the translation done and have a place where I can always recognice which code version is running currently. I know that you need the code editor level for all the flexibility and stuff you can change - it was more a question about the term "actions" but I guess you call that user selector / swicht an action.
For me it is important to get such Attention call in just 1 line and german language tends to be long(er).

Before I read this recent messages I had already created a picture of a more structured configuration page which I will add in another comment before I will try out the latest code as you seem to have announced it in the latest comment but did not mention what version that should be. But I will try it out.

**Comment by typxxi** [comment]
Here is the configurator page how it could like incl. the TOP Message regarding that things will not be updated or changed before the configuration will be saved. You can see that I started with that particular line heightwise at the level of the right preview so that it looks nicer.

ATTENTION: I did only picture manipulations so all the old text boxes needed the updates announced / described in the translations. 

The new lines I had added use of cause different fixed width fonts and sizes so I can not judge if that might fit or if I have to shorten things . It is just to give you an idea how it could look like without the need for a long description cause UI descriptions are long, boring and cause a lot of faults unless you replace those by a picture like this one.

![image](https://user-images.githubusercontent.com/20662430/122853359-64444580-d312-11eb-9c35-bc1feb7cba27.png)

And as said before now it is usefull to have the translation cause the 4th switch description is completely outdated from the very first translation mentioning "body values" and "switch" were body values is completely off.  
"Power button and User switches" would be and fit perfectly even though on friday you might have found the perfect unfold / collapse toggle and how that should be called.   

But next DEV version should have these translations cause Körperwertanzeige is terrible to read in that context and did not /never fit and I would never have used it if I had known in which visual context it would have been used.

Now i will download the new release and hope the best.

**Comment by typxxi** [comment]
**ATTENTION:** This one here is already updated and therefore you can please reduce your opener to a short introduction and link to this comment just simply to avoid 2 such big and 99% overlapping entries - your opener is more or less outdated by this.

In this one here I will also mark points as **completed** and mention its release or its **progress from now onwards**. 
Plese note that the next chapter title will stay with its previous LATE NIGHT BUILT 0620 cause that is where it all hat started.
Finally I will resort the items if a topic is done or postponed in a sense usefull but not yet possible or not top priority, but the once given number will stay the same. 

EVALUATION OF LATE NIGHT BUILT 0620 
=============================
First note goes to the built number: 0620-hh should be the new sheme where hh are representing the hour  of release cause I had yesterday only gotten 0620s

I will start with pure translation points except that I wanted to start from top to bottom and therefore begin with a non translation issue but a more general one which you had already answered and I added your reply as state: not possible yet.

I. TRANSLATIONS
=============================

[0622-01]: 
**PLEASE ADD THE GERMAN TRANSLATIOLINES FOR #5 + #7 in THE NEXT RELEASE JUST TO LET ME SEE HOW THEY LOOK IN GERMAN in the UI context** regardless how far your progress will be till then regarding a power button replacement cause even right now it would still fit in as that button is stil the power button.

**#5** 
Search for 
`"Detaillierte Messergebnisse anbieten" `
and replace the whole german term by this
`"**Detaillierte Messergebnisse anbieten (untere Hälfte - per Power Button einblenden)**"` 

This means "Offer detailed measurement results (lower part - press power button to show)"

As said before it would be nice if there is a way to add a carriage return line feed / new line after "... anbieten" but that does not have the highest priority to get it done, just looks much cleaner.
`"Detaillierte Messergebnisse anbieten`
`(untere Hälfte - per Power Button einblenden)"`
I had seen that before but I can not tell  you where.

**#7**
Search for 
`"Symbolleiste"`
and replace the whole german term by this
`"Symbolleiste anzeigen (Power Button und Schalter)"`

I had to change the terms inside to "Power Button" (the translation "Einschaltknopf" sounds weird in that context cause it would mean to turn the whole scale on / off which it does not do). 

Therefore I still suggest to use any different kind of button instead that particular turn on / off button which will cause questions.

I still recommand an icon that represents to hide / unfold something like a triangle or twister ᐅ to press for more details and if that had been pressed I suggest to replace that previous ᐅ with the opposite twister / triangle like this ▼ cause this switch has a toggling meaning to show or hide details. 
A power button on the other side would always target on a device as a whole like turn on / off your complete HiFi not a particular feature on that stereo. 


Fix the fix = former #2
========================
This "chapter is for all those tiny changes / mistakes needed after a quick release like in this case a "copy and paste" mistake cause you had taken the english term which does not fit in german.
You simply have to replace 1 LETTER by 2 to get it fixed from "STATE: unbekannt" to "STATUS: unbekannt"
![image](https://user-images.githubusercontent.com/20662430/122860811-affcec00-d31e-11eb-8611-8e74969f23aa.png)


**#6**
**I have now updated this LINE again to explain its purpose better like "offer account buttons (lower right)" where buttons is the right term cause you can simply click 1 at any time.**

Search for 
`"Schaltfläche anzeigen" `
and replace the whole german term by this
**`"Konto Auswahlbuttons anbieten (rechts unten)"`**

Previous "Schaltfläche" means a (bigger) switch area like 'cancel' or so: 

My personal suggestion is to start with 5 predefined buttons from 1 to 5 which everyone can change in the editor to his taste. 
Button "letter" "1" to "5" will fit best for all I guess cause everyone familiar with can replace the 1 with his initial letter or the one he wants to get. Also: Consider to use ONLY the "mdi:alpha-c-circle-outline" cause these are better and easier to read cause I had asked my dad and mom and a relative too that the outlined chraracters and numbers are easier to read at least here for people 45+. But at least we should mix letters and numbers to show the options that are available here.
And finally you should add an empty line below the SWITCHES cause now the BMI line comes far too close to the switches. Does not look as good as it could. Therefore try to add an empty line below the buttons and before the BMI line.
![image](https://user-images.githubusercontent.com/20662430/122862730-e0925500-d321-11eb-8a76-b5a1ea09eb0b.png)


**[0622-01]: #8 = REOPENED due to NEW REQUIREMENTS**
Search for 
`"Bitte beachten ...."`
and replace the whole german term by this
`"ACHTUNG: Weitere Optionen sind nur im Code Editor verfügbar. [0620-hh]"`

**[0622-01] 2 NEW REQUIREMENTS:** 
#1 Could you limit the BOLD to "ACHTUNG:" only ?

#2 And then please replace the further line with this shorter one below to get the whole piece in just 1 line, not 2. 
Therefore reduce the version number to "*-hh" instead of "*-hh:mm" cause ":mm" will not really help us!
We will not be able to release 2 versions within 1 hour and even if that ever might happen  once or so you could easily increase that particular "hour counter" by 1 cause when you will release a new version I will usually not install it so fast, that I can report a bug to you that you will then be able to fix and release in a new version - all of that in just 1 hours. Even if we will achieve that you can simply give it a 02 even if it is still 01 oclock. Adding a ":20" makes it harder to recognice and check without getting any further benefit.

"**ACHTUNG:** mehr Optionen nur im Code Editor verfügbar [0620-01]"
"**ATTENTION:** more options available only in the Code Editor [0620-01]"  
which had been before 
`"ATTENTION: Further options are only available in the code editor. [0620-hh]"`

QUESTION: I have deleted the word "actions" cause I was not sure if there are any actions available or not and if actions have to be mentinoned there cause usually it will cause questions like "What actions ?" 
I'd prefer the short and simple one cause that would work better on tablet or smartphones too and would not lead to ask what other "actions" are available. 

**[0622-01]: #9 = REOPENED due to NEW REQUIREMENTS**
This translation in the beginning of the former translation file has been lost cause the translation had been already there and is now replaced by the oldest  or first one. 
"name": "BodyMiScale Karte",
"description": "Die BodyMiScale Karte kann Ihnen Ihren gewichtsmäßigen Körperstatus anzeigen."

**[0622-01] 2 NEW REQUIREMENTS** 
#1 for a shorter and more precise message and 
#2 for the dev version in the title the [DEV 0622-01] as you had done it before-

Please replace the just brand new updated
`"description": "Die BodyMiScale Karte kann Ihnen Ihren gewichtsmäßigen Körperstatus anzeigen."`
with this one 
`"description": "Die BodyMiScale Karte zeigt Ihren gewichtsmäßigen Körperstatus an."

`"The bodymiscale card shows you your weight related body status"` compared to the previoius
`"the bodymiscale card could present to you your weight related body status"`
just short and straight without any "could" or "present to you your" overhead not really needed there.

Compare the first release to the latest 0622-01 and you recognice the difference 
![image](https://user-images.githubusercontent.com/20662430/122860550-349b3a80-d31e-11eb-8bfa-14c493a47ce9.png)


**POSTPONED till it is possible for custom cards too: #1**
=======================================
card configuration title should be 
`"Lovelace body miscale Card Configuration"`

For me it should be the spot #1 for the BUILT number like cause I was struggling a lot what code my card was relying on which is very frustrating to not be able to get behind which code is running except by differentiating that due to its translation errors  
`"Lovelace body miscale [0620-hh] Card Configuration"`  
STATE: post poned cause right now not possible


**[0622-01]: #2**
A new error or untranslated term had appeared behind the scale icon "unknown" 
![image](https://user-images.githubusercontent.com/20662430/122768868-4f30cd80-d2a4-11eb-9656-140cc5470553.png)

I guess we need to call that "STATUS: unbekannt"  (means "STATE: unknown")
Therefore the previous "OK" value has to change too into "MESSUNG: OK" (means "MEASUREMENT: OK"). 
That way both messages are accurate for its purpose, first for scale in general, second is about the measurement results.

Search for 
`"OK"`
and replace the whole german term by this
`"MESSUNG: OK"`

And regarding "unknown" you have to implement that on your own as "State: unknown" - for german "STATUS: unbekannt"
Do not forget in case of errors to show behind the scale icon the state message "State: error" - for german "STATUS: Fehler", at least as long as both icons are shown at the same time. 


**[0622-01]: #3**
Search for 
`"Auswahl der Waage / Person erforderlich"`
and replace the whole german term by this
`"Bitte ein Waagen Konto auswählen [erforderlich] !"`

After being unsatisfied with the previoius term (which is right but feels weird to read and explain) we now could go the technical way which would mean a "Sensor - user name" combination or much easier "Please choose your scale account [mandatory] !"

You have a scale and even if you are alone the system it can only work if you have created at least 1 account in that bodymiscale.yaml file - can you tell me that a second user has to be added in that file too ?


**[0622-01]: #4** 
Search for 
`"Bild (Optional)"`
and replace the whole german term by this
`"Hintergrundbild [optional]"`

For me it is not about "pic" but about a "background pic" cause the figures will lay in front that (background image). 


II. GENERAL CONFIGURATION
=============================

This chapter is beyond translations, more integral questions in general like in which order why option should appear or even been hidden cause it makes no sense to offer switches that will not be shown if the general symbol bar is off, but that explanation will follow below.

Just an example that you first need a symbol bar that is turned on to then make those other switches available.

**I.**
You need a new first line as explanation of the general behaviour
BIG LETTERS: CHANGES WILL ONLY APPEAR AFTER BEING SAVED.

Why is that important ? 
I had struggled with it but only in certain cases. While in configuration you can press any of those 6 switches but nothing will change before you will have saved it. But on the right half iof the configuration dialog you have a preview and for example that power button in the preview will be working regardless how the switches are switched or not.  

**II.**
Please move that "model number" input line above that "Background image" line for the simple reason that all mandatory inputs should get the highest amount of attention in the top section and should be selected first followed by the less important optional ones like that Background image.

**III.**
I would love to see a better structure in the configuration especially regarding these in total 6 switches.

![image](https://user-images.githubusercontent.com/20662430/122775510-7ee2d400-d2aa-11eb-8f16-0cc8f7dc1ef1.png)

Therefore can you please add an empty line (or separator line out of ----) above the first switch which is followed by this line
```"Options for card header" = "Kartenkopf Optionen"```

**IV.** 
After these first 3 switches I would love to see 
an empty line
followed by this additional  
`"Further card options (measurement details / switches)" or in german "Weitere Kartenoptionen (Messung Details / Schalter)"`
Yes, both german versions are this time shorter !

It will make it a lot easier in the beginning to understand where the targets of the upper half are and what each switch is for while we both do it know now blindly due to daily training but the average user will struggle which swtich is what for (therefore I had pointed out where those things are located).

**V.** 
Right now the switches 4, 5, 6 are not in the RIGHT and an INTUITIVE order !
`4 show button for more measurement details `
`5 show switch`
`6 show symbol bar`

But that LAST SWITCH "show symbol bar" has to be the first choice in the lower section cause **without showing the symbol bar** switches 4 and 5 are useless to click on and therefore should stay hidden / unavailable, right ?

Only once "symbol bar" is activated the other 2 switches should appear cause only then they can work.
Without icon bar enabled the "show button" and "show switch" are running into the emptineess as we called it here.
`4 Show symbol bar`
`5 Offer button for more measurement details` 
`6 Show switch`

I chose "offer button for more measurement details" cause you only offer a button which has to be pressed to show further measurement details. This is not a general switch to always show the measurement details. "show" instead "offer" will be misunderstood cause if you would activate that switch "show button for more measurement details" people will mostly only read and expect "show more measurement details"

**VI.**
The **"Switch (U)"** should get a title like **"Show scale account selector(s)"** and that would finally mean to replace "show switch" by "Show scale account selector(s)" ? 
ATTENTION: if you already have done the translation tasks mentioned above in the first section then you will have to do this

Search for 
`"Schalter (U) anzeigen (rechts unten)"` 
and replace the whole german term by this
`"Den / Die Waagekonten Auswahl Schalter (U) anzeigen (rechts unten)"`

**VII.**
Regarding the **model number input** I was not happy too and I am not convinced about that way you had implemented cause in reality it is - at least right now - just a choice between 2 scales and nothing more. Therefore: 

**Isn't a switch a better solution to chose between 2 models where '181D' should be the default setting?**
I suggest as title "**Do you own model 181B incl. impedance measurement?**"

By default it is turned of and therefore it will run as a model 181D user. 
Everyone with a better model does not loose much with such basic setting and can switch easily to the better 181B to get more measurement details.
If the "poor" 181D owner turns on this switch for the 181B model he will get a lot more label but no values and learns what the differences are between those similar looking scales - as I had learned it too from you and your code / data. 

That typing solution is not ideal and it would never be my first choice even though you read the initial value out of the yaml file and you might write that value back too. Such switch would work at least right now a lot better and looks nicer too for sure.
 
**AND:** a **switch does not need the additional "mandatory" term** too cause with a switch the card would start by default as 181D.

**VIII.**
I guess we have to change the order of the detailed measurement results in the lower half.

BMI should be followed by 
BMI classification
FINALLY: Age can be deleted from that measurement detail page cause it is not really needed there.  

III. POINTS TO NOT FORGET
=============================

**1. VARIABLE OR LABEL NAME with TYPO**
I saw in the translation file this "Lack-exerscise": "Bewegungsmangel" and exercise is spelled without the s in "rsc"

**2. ALL ABOUT FIGURES AND FORMATS**
* usefull, consistent decimals like 2 decimals for weight
* metabolism kcal figure with national / domestic thousand separators like german 1.856 kcal instead of 1,856 kcal
* central european figures with a "," as decimal separator cause 102.3 is not right here
* long term: user CONFIGUREABLE DECIMALS in bodymiscale.yaml

**3. Aligned lines for Status (upper left)**
For the left upper status we should make sure that the users gets just 1 icon (scale or warning, but never both cause they are contradicting) in case things are going wrong like for example in a measurement.
181B owners can get longer and misaligned error messages that go over 2 lines where the 2nd line starts
even below that warning icon instead of being aligned to the first line.
![image](https://user-images.githubusercontent.com/20662430/122782583-219e5100-d2b1-11eb-87ce-00f17f2ae166.png)

**Comment by typxxi** [comment]
Just to make things clear: 

Before today I had not really immersed into that "scale sharing" or "multi user scale" usage thing cause our kids / girls are too old to share that and my wife and I have each our own scales cause I am used to bathroom usage and she to bedroom / cabinet / fitness for what ever reason even though I did not know that she was using my sacle in the bathroom too but she is half the size or sub 60 kg versus 95 - 105 kg. 

That being said today was the first time that we both together just went throught the UI again and this had changed a lot cause these are 2 different worlds - you alone or 2 to 5 in a "multi user environment".

And then things changed but before I write it all down I simply have done a screenshot that you see what I mean first and then get the words to copy and my english explanation why cause you have to manage french and english too.

![image](https://user-images.githubusercontent.com/20662430/123172602-a4740700-d47d-11eb-8e9d-6b72e6bddd00.png)

I now use the number of the switches from 1 to 7
the second line is the new german title (I did not copy the phrase with the () cause these are right 

#1 
Einschalten für Model 181B ! 
turn on if you have model 181B and germans use ! in case of the impetus / command . this is a lot shorter and everyone who wants to use the scale needs to know what he owns ... I have only 1 consideration I had forgotten or over seen when I came up with the idea. What if this whole thing shall support more than 2 different models when 181Z might be released ? That would need more space and a different button to choose 1 out of 2 or 3  ... just a carefull reminder that switch is great for now, but maybe a dead end in case model 181Z comes out - but I tell you that I have found a third and more logical and usefull solution cause my idea "use a switch" was good, but not well thought enough cause from the mandatory top position it went downwards and at the end of this comment you will find 1 solution that would bring this topic on its former position as number 2.

#2
Name des Waagekonto anzeigen ?
"Besitzer anzeigen", oh what a great idea - and what a crap 5 days latter, sorry, but "show owner" does only work in a single user case but it has to be multiuser UI
  
I call it now "show name of the scale account" cause in any multi user environment for such a scale we need to have scale acount and each get a name ( I personally have abused that a bit cause I added a kind of title to it "Gewichtsstatus Wolf" "weight state Wolf") 

#3 OK

#4
persönliche Stammdaten anzeigen ?
this literally means "show personal base data" cause base data alone is not enough at least in a  multi user environment

#5 OK

#6
Waagekonten Umschalter anbieten ?
Ok, that is "offer scale account toggles ?" cause that's what the 5 switches are / will be for 

#7
ATTENTION: you simply have to replace just "Schalter" for "Umschalter" (cause a switch has 2 states and a "Umschalter" is 1 out of 5)

Please move line 7 upwards onto the position of switch 5 whitch then becomes number 6  and so on.

Helps a lot regarding structure to separate it cause my wife told me if  I was crazy that the last one is responsible for 5 and 6 cause she had played with it on her own cause she had arrived earlier. 


ATTENTION: GERMAN CARD DESCRIPTION MUST CHANGE please.
==============================================
FINALLY and most importantly: 

#1 
please add the DEV [0622-22] here in the selection and change the description cause that one is outdated and I always have to ask myself what is what for - and of cause my wife went crazy too.
![image](https://user-images.githubusercontent.com/20662430/123176881-91186a00-d484-11eb-94c8-ab645a84e660.png)

#2
I have change that body scale card description earlier and here is what it should be if you agree cause the version in the beginning  was too complicated without additional informations.

"Die BodyMiScale Karte zeigt Ihren gewichtsmäßigen Körperstatus an."
"The bodymiscale card shows you your weight wise / related body status"

Here in germany the whole topic is delicate and we have to limit the "competence" of such scale to a "weight" wise related body status and not healthwise = medical body status. But that might  be a cultural thing and the average french is most likely not as fat as germans .

#3 
As my wife reminded me just a minute a go we need that single line on TOP of that configuration page as shown in my screenshot composition 2 days ago: 
"CHANGES WILL ONLY APPEAR AFTER BEING SAVED."
"ÄNDERUNGEN WERDEN ERST NACH DEM SPEICHERN SICHTBAR."

She had found a case where some things changed instantly while others not like Power Button that works in preview while others do not work. 

#4 
The 181B and 181D question belongs to the top and while watching at the previous design here i got the idea to bring the the model question back to the top 2 parameters where she belonged as a mandatory decision that belongs directly below the scale-account selection: use the scale-account selection shemata for a choice between 
* 181D (default)
* 181B (Bio Impedance)
* others (just in preparation of any future scale that might work that way too) 
![image](https://user-images.githubusercontent.com/20662430/123178178-f2d9d380-d486-11eb-8820-612204eb93a9.png)

And if that model selection would work the way like the scale user account selection we would be back to the previous structure and 6 switches for questions which could also be separated into the bing ones: 1. header and 2. bar / switches like here 
(ATTENTION: the words and sentences are maybe not at the latest stage , but I guess you get behind the advantage of the structure. 

![image](https://user-images.githubusercontent.com/20662430/123179297-5d8c0e80-d489-11eb-8fad-c4ba42c40334.png)

**Comment by dckiller51** [comment]
I'm looking at your changes this morning. Be careful for the scales, there will be no other model to add because it is linked to the custom components so ultimately your idea of ​​specifying just a switch with impedance or not is the best solution.

**Comment by dckiller51** [comment]
> Just to make things clear:
> 
> Before today I had not really immersed into that "scale sharing" or "multi user scale" usage thing cause our kids / girls are too old to share that and my wife and I have each our own scales cause I am used to bathroom usage and she to bedroom / cabinet / fitness for what ever reason even though I did not know that she was using my sacle in the bathroom too but she is half the size or sub 60 kg versus 95 - 105 kg.
> 
> That being said today was the first time that we both together just went throught the UI again and this had changed a lot cause these are 2 different worlds - you alone or 2 to 5 in a "multi user environment".
> 
> And then things changed but before I write it all down I simply have done a screenshot that you see what I mean first and then get the words to copy and my english explanation why cause you have to manage french and english too.
> 
> ![image](https://user-images.githubusercontent.com/20662430/123172602-a4740700-d47d-11eb-8e9d-6b72e6bddd00.png)
> 
> I now use the number of the switches from 1 to 7
> the second line is the new german title (I did not copy the phrase with the () cause these are right
> 
> #1
> Einschalten für Model 181B !
> turn on if you have model 181B and germans use ! in case of the impetus / command . this is a lot shorter and everyone who wants to use the scale needs to know what he owns ... I have only 1 consideration I had forgotten or over seen when I came up with the idea. What if this whole thing shall support more than 2 different models when 181Z might be released ? That would need more space and a different button to choose 1 out of 2 or 3 ... just a carefull reminder that switch is great for now, but maybe a dead end in case model 181Z comes out - but I tell you that I have found a third and more logical and usefull solution cause my idea "use a switch" was good, but not well thought enough cause from the mandatory top position it went downwards and at the end of this comment you will find 1 solution that would bring this topic on its former position as number 2.
> 
> #2
> Name des Waagenkonto anzeigen ?
> "Besitzer anzeigen", oh what a great idea - and what a crap 5 days latter, sorry, but "show owner" does only work in a single user case but it has to be multiuser UI
> 
> I call it now "show name of the scale account" cause in any multi user environment for such a scale we need to have scale acount and each gets a name ( I personally have abused that a bit cause I have added a kind of title to it "Gewichtsstatus Wolf" which means a title "weight state Wolf")
> 
> #3 OK
> 
> #4
> persönliche Stammdaten anzeigen ?
> this literally means "show personal base data" cause base data alone is not enough at least in a multi user environment
> 
> #5 OK
> 
> #6
> Waagekonten Umschalter anbieten ?
> Ok, that is "offer scale account toggles ?" cause that's what the 5 switches are / will be for
> 
> #7
> ATTENTION: you simply have to replace just "Schalter" for "Umschalter" (cause a switch has 2 states and a "Umschalter" is 1 out of 5)
> 
> Please move line 7 upwards onto the position of switch 5 whitch then becomes number 6 and so on.
> 
> Helps a lot regarding structure to separate it cause my wife told me if I was crazy that the last one is responsible for 5 and 6 cause she had played with it on her own cause she had arrived earlier.
> 
> # ATTENTION: GERMAN CARD DESCRIPTION MUST CHANGE please.
> #1
> please add the DEV [0622-22] here in the selection and change the description cause that one is outdated and I always have to ask myself what is what for - and of cause my wife went crazy too.
> ![image](https://user-images.githubusercontent.com/20662430/123176881-91186a00-d484-11eb-94c8-ab645a84e660.png)
> 
> #2
> I have change that body scale card description earlier and here is what it should be if you agree cause the version in the beginning was too complicated without additional informations.
> 
> "Die BodyMiScale Karte zeigt Ihren gewichtsmäßigen Körperstatus an."
> "The bodymiscale card shows you your weight wise / related body status"
> 
> Here in germany the whole topic is delicate and we have to limit the "competence" of such scale to a "weight" wise related body status and not healthwise = medical body status. But that might be a cultural thing and the average french is most likely not as fat as germans .
> 
> #3
> As my wife reminded me just a minute a go we need that single line on TOP of that configuration page as shown in my screenshot composition 2 days ago:
> "CHANGES WILL ONLY APPEAR AFTER BEING SAVED."
> "ÄNDERUNGEN WERDEN ERST NACH DEM SPEICHERN SICHTBAR."
> 
> She had found a case where some things changed instantly while others not like Power Button that works in preview while others do not work.
> 
> #4
> The 181B and 181D question belongs to the top and while watching at the previous design here i got the idea to bring the the model question back to the top 2 parameters where she belonged as a mandatory decision that belongs directly below the scale-account selection: use the scale-account selection shemata for a choice between
> 
> * 181D (default)
> * 181B (Bio Impedance)
> * others (just in preparation of any future scale that might work that way too)
>   ![image](https://user-images.githubusercontent.com/20662430/123178178-f2d9d380-d486-11eb-8820-612204eb93a9.png)
> 
> And if that model selection would work the way like the scale user account selection we would be back to the previous structure and 6 switches for questions which could also be separated into the bing ones: 1. header and 2. bar / switches like here
> (ATTENTION: the words and sentences are maybe not at the latest stage , but I guess you get behind the advantage of the structure.
> 
> ![image](https://user-images.githubusercontent.com/20662430/123179297-5d8c0e80-d489-11eb-8fad-c4ba42c40334.png)

Update 0624-10  Available

**Comment by typxxi** [comment]
Sorry Aurelien,

I had been working on this while you were working on the next release I guess.
As you can see it took 1,5 hours to get that picture right and then I saw your reply Update available just 1 minute ago when I wanted to comment. So I added these 2 lines right now and you can see below my thoughts and the picture how it could or should like ... if possible. Maybe at some point this option will disappear if you will find a detection of what scale is connected as you had said somewhere. 
I will now move on thoe your 0624-10 but this below is state of this morning and your previous 0622-22 release.

> I'm looking at your changes this morning. Be careful for the scales, there will be no other model to add because it is linked to the custom components so ultimately your idea of ​​specifying just a switch with impedance or not is the best solution.

Well, 
somehow it sounds like the best in case of just only 2 scales to choice from.

BUT: that is only true for both of us or all those that know that there are only 2 scales supported, cause this kind of switch can only lead to ask for the better one, the 181B and does not allow to mention the second one.

AND: This is a requirement like the choice of a scale account, so it need to be here in a kind like this and then below the background image as an option.

![image](https://user-images.githubusercontent.com/20662430/123232555-8fc55c80-d4d9-11eb-9259-494906a50108.png)

**Comment by typxxi** [comment]
> Update 0624-10 Available

wowwww .... looks beautifull

and now I was able to update in 3 minutes instead of hours. I will look into that now and let  you know later.

**Comment by dckiller51** [comment]
Look
![Uploading Screenshot_2021-06-24-11-01-18-811_io.homeassistant.companion.android.jpg…]()

**Comment by typxxi** [comment]
I     LIKE   IT    A    LOT  ... still pretty impressice leap or jump today

I will continue to show the picture first and then describe the changes needed in german and the english explanation that you can stay in sync.

![0624-10 - for 0624-12](https://user-images.githubusercontent.com/20662430/123241601-95bf3b80-d4e1-11eb-9773-905297d25255.jpg)

I guess we both agree that the entity is a combination of scale and account on that particular scale and that till now each of these scale account terms sound ugly or weird and I had to search for a better, more self explaining one which I finally had found.
 
#1 
'Bitte ein Waagen Konto wählen ... '
There are 2 changes needed:
1. this description and line has to be in a taller fontsize cause it is 1 of 2 requirements which needs attention and deserves size
2. Translation "Bitte ein Konto auf der Waage wählen (erforderlich) !" 
This means "Please choose an account on the scale (required) !" 
The previous one was too complicated and this one seems more self explaining and intuitive to choose an account on your scale in case it is used in a 2 or more person household

#2 
these 2 "headlines" over the switch batteries  need to be taller as a "counterweight" to the massive switches below just due to the big size of those switches. Now they are too small, feels not right, cause a headline have to be bigger.

#3
the translation has to change due to logic mentioned in #1. 
Now it is longer, but again it 'feels' more selfexplaining, logical, precise about the whole: what to show and where - or not.

"Namen des Konto als Titel anzeigen ?"
"Show the name of the account as title ?" 

#4
The first word has a TINY typo. Just replace that 's' with a 'c'  - it had to be "persönliche"

#5
This is another consequence of #1 and I took it to a meta level getting away from the deep down technical level talking about showing buttons or not. This should be 
"Kontowechsel erlauben"
"allow to switch the account" which means that it will offer the possibility to switch the account cause only 1 account can be used at any given time / measurement. 

#6
This is at the end a simple change, but longer one to explain. Therefore I will break it up into single steps

1. you have to SWITCH the titles of the last switch ("Symbolleiste " or "icon bar") with the "Detaillierte ..." cause 'icon bar' has to be the headline as that is your first decision you have to make. The RED DOUBLE ARROW shows what I mean and the number 6 with its 2 pointers.

2. The current title "icon bar" does no longer feel right! 
An end user in a family would not expect that you will find below the headline 'icon bar' such major features burried like 'more measurement details' and 'account change'. 

Therefore it is CAN NO LONGER BE an 'ICON BAR' (which is just a technical term being far away from the end user perspective like my wife). My suggestion is 
"Advanced options (account change and more measurement details)" 
"Optionen für Fortgeschrittene (Kontowechsel und weitere Messergebnisse) 

We do no longer need to mention a 'power button' or 'toggle'  if it is about 'feature sets' to allow 'account change' or to 'show more measurement details' - that is what users wanna know. The will press for sure on the buttons and find the right ones.

I guess these points are easy to fix so far even though a bit more considering that you have to keep an eye on french and english version too.


FINAL POINT
My last point has no number in this screenshot. It is about the scale model and it can be broken down into 2 steps.

1. "Besitzen Sie das Model inkl. Impedanzmessung?" is too long and to complicated. 
And even a model 181B question will not really be self explaining, but I have found a better solution.

"Turn on if your scale has 4 grey circles of 5 cm  Ø on top of it (model 181B) !" 
No doubt to misunderstand that except someone wanna play with that option and see if they could get more out of a 181D by turning that option on. Here is the german translation
"Aktivieren, wenn oben auf der Waage 4 graue Kreise von je 5 cm Ø sind (Modell 181B) !"

2. But that solution above would be my last chocise cause this would always be my first spot and choice as long as it is a basic requirement that has to be chosen first. 
And at the current spot it is not right there and does not have the size it needs while the other switches for examples do have a headline and this one does not. 

Maybe this one might convince you about the direction and how much cleaner it looks

![image](https://user-images.githubusercontent.com/20662430/123253053-4e3eac80-d4ed-11eb-85a7-341ce8ac3860.png)

Beyond that I have a final point but that is a different story about the underlaying opportunities for individual improvements I will put into another comment that has nothing to do with the current UI / setup page

**Comment by typxxi** [comment]
If you want to change the Card's title you have no easy chance to access it and change the headline for example.
You have only a toggle: headline on or off

But my wife was not really happy with that and I not either. Therefore I used the friendly name but that can not be part of the bodymiscale.yaml file cause that will lead to errors but I added a comment that explains how additional data for sensors do work in general.

And I guess it is usefull for all users to get in touch with it this way as long as everybody has to deal with this file and its structure. You do not need to add much to the readme cause everything to know is written in there and self explaining too at least to a high degree that you can start to play and tinker around what a sensor picture is (tiny round shaped picture which makes it a lot easier to identify and select the right sensors in general) and how to deal with friendly names for sensors that are created by other integrations and are not part of the configuration.yaml

In my opinion the best way to train people a bit and give them an opportunity to learn more about and tinker a bit. 
At least it does not hurt and opens the horizont and that way people do not forget a lesson they had learned.

```
wolf:
  sensors:
    weight:       'sensor.ble_weight_mi_scale2'
    # impedance:  'sensor.impedance_aurelien'
  height:         200                         ###   avoid '' or "" cause it is not a string, but a number
  gender:         'male'
  born:           '1979-11-11'                ###   'YYYY-MM-DD'
  model_miscale:  '181D'                      ###   '181D' (scale has not the 4 dots on top / Bio Impedance) or '181B'         ###
  ################################################################################################################################
  # These usefull attributes do not work this way here in this file. They have to be added in the [customize.yaml]
  # friendly_name:  'GEWICHTS STATUS Wolf'      ###   the title of the sensor usefull for cards etc.
  # entity_picture: '/local/img/miscale2.jpg'   ###   small icon for the sensor usefull if used in other basic cards in $config/www/img/
  # icon:           'mdi:scale'                 ###   icon of the sensor
  ################################################################################################################################
laura:
  sensors:
    weight:       'sensor.ble_weight_mi_scale2'
    # impedance:  'sensor.impedance_aurelien'
  height:         178                         ###   avoid '' or "" cause it is not a string, but a number
  gender:         'female'
  born:           '1987-07-07'                ###   'YYYY-MM-DD'
  model_miscale:  '181D'                      ###   '181D' (scale has not the 4 dots on top / Bio Impedance) or '181B'         ###
```

**Comment by dckiller51** [comment]
I could make the modifications only this evening but these 2 points ask me question # 3 The title is modifiable, we can put whatever we want so I think we have to change the title.
# Final point I think that we should not refer to the balance model but to the impedance because for example for my wife I will not activate the option. Because to have the impedance we must stay several seconds on the scale then when it is weighed the impedance is at -2 it goes down sooner and does not wish to have this data.

**Comment by typxxi** [comment]
> I could make the modifications only this evening but these 2 points ask me question
> #3 The title is modifiable, we can put whatever we want so I think we have to change the title.
> #Final point I think that we should not refer to the balance model but to the impedance because for example for my wife I will not activate the option. Because to have the impedance we must stay several seconds on the scale then when it is weighed the impedance is at -2 it goes down sooner and does not wish to have this data.

Sorry, I did not read this till now.

If you refer with #3 to this 

#3
the translation has to change due to logic mentioned in #1.
Now it is longer, but again it 'feels' more selfexplaining, logical, precise about the whole: what to show and where - or not.

"Namen des Konto als Titel anzeigen ?"
"Show the name of the account as title ?"

That is just about showing the title / name of the account. 
I has only modified the friendly name to get what I thought looked good - for me. 
I did not thought about it in general, just about a way that others can use too. 

Sorry, but I am really not sure what points you mean so it is a bit of guess work and might been pretty off what you wanted to know from me. 

What is the balance model ? I thought the 181B is the bio impedance model, the more expensive one.

My description in the bodymiscale.yaml was just an expample, I have not thought about balance nor knowledge about them, just happy to know the difference by picture and its 4 big dots where we have a just clean white surface. 
I also have updated that file in the model_miscale line which was partly false or misleading

**Comment by typxxi** [comment]
Updates for 0625-01
![image](https://user-images.githubusercontent.com/20662430/123369168-8ab9e900-d57d-11eb-984d-1f4e2d1c37ac.png)


#1
The font size for the most important selection is too small, that 'Bitte ein Konto auf der Waage wählen ....' and background Image too.

Could be that you can not change it but then let me know if that is not possible.


#2
"Aktivieren, wenn oben auf der Waage 4 graue Kreise von je 5 cm Ø sind (Modell 181B) !"
"Turn on if your scale has 4 grey circles of 5 cm Ø on top of it (model 181B) !"
No need to ask for a cumbersome tiny small model number hard to find on the scale while it is a lot simpler to ask for 4 grey 5 cm circles on the white surface and turn of if the scale has those 4 circles.
Right now there is the question for "impedanzmessung" and so on which is far too complicated compared to 4 circles or not !


#3
2nd headline "2. mehr Kartenoptionen": Remove "(Messungsdetails / Schalter)" cause these details are no longer needed


#4
the 'ICON BAR' is no longer right and should become
"Optionen für Fortgeschrittene (Kontowechsel und weitere Messergebnisse)
"Advanced options (account change and more measurement details)"

I know that the buttons are not supported, but we should let it stay that way it was meant to be used earlier. 


#5 
Even though we might not use it actively this title has to become 
"Kontowechsel erlauben"
"allow account switch " 


#6
This is an easy one: you simply have only to change the order of these last 2 switches at the bottom cause "offer measurment details" or "Detaillierte Messergebnisse anbieten ..." has to be the first option to choose from since "account change" will loose its role.


And FINAL TASK might be the "model question" style as part of the required ones in the top instead of a switch.
![image](https://user-images.githubusercontent.com/20662430/123369714-8b06b400-d57e-11eb-82ad-a5fee79d8fb7.png)

So far the scale works fine here - but we are still in 'separate' mode which means each one of us is using her / his scale and also the measurement results arrive in our MiFit App and also in HA in the right card for each of us.
So far great progress and not that much to change.

thx

**Comment by dckiller51** [comment]
#1 The size is defined by hassio. All cards have this size for this type of box

Update ok 0625-09

If ok I'll be able to realease the official version of HACS.
Next step replace the power button with a chevron

**Comment by typxxi** [comment]
Hi Aurelien,

here are my comments for 0625-09 as the final step before release I guess. 
Again first the picture with a #number for all the issues I have found to get things right this time.
Be aware that #4 and #5 have gotten a mjaor issue cause now these points have been "mixied together".

![0625-01 - 1 = fixes needed](https://user-images.githubusercontent.com/20662430/123427598-1d31ab00-d5c5-11eb-97c1-66caf354acf5.jpg)


FIRST OF ALL:
=============
**Each switch needs a decision / answer** and therefore we **need a consistence type of actions / questions** and have to end each with `" !" or " ?"`

We want answers / decisions and then we have to add those `" !" or " ?"`
Will look a lot better to have **clean and clear messages** what to do.

Here are the "german" and "english" translations for your copy and paste job.

#1
ATTENTION: THIS IS A "NEW FEATURE" 

Please add "KONTO: " ahead of the current sensor title / friendly name
"KONTO: "
"ACCOUNT: "
That will make the UI / configuration page consistent to the result cause "account" is king to understand a scale.


#2
Copy and paste  cause these are minor changes, but it is **important to emphasize** what has to be done by capitalizing the core message **"ACTIVATE, if"**
"**AKTIVIEREN**, wenn auf der Waage 4 graue, 5 cm Ø Kreise sind ( = Modell 181B) !"
"**ACTIVATE** if the scale has 4 grey circles of 5 cm Ø on top ( = model 181B) !" 


#3
Please **add space above each of these 2 headlines** or simply add an empty line above those to give each  more space they deserve as they are headlines of  a kind of chapter


#4 and #5 
Here things might have messed a bit and have been mixed together. Therefore we have to bring them back in right order 

#4
"Fortgeschrittene Optionen anzeigen ?"
"Show advanced options ?"
no further details needed ... and no word about icons or switches as those are described below !

#5 
"Weitere Messergebnisse anbieten
(untere Hälfte - per Power Button einblenden) ?"

"Offer further measurement details
(lower half - power button will show those) ?"

#6
**FINAL CHECK:** there are **3 more switches I have not mentioned above in the translations which all need that ' ?'** at the end so please add them now before they might be forgotten !

I hope that this is it and that you can tell me how I can get access or switch to that **english UI** to have a final look on that too before released.
Is it just up to me to change the language in HA settings in general or is there a trick or feature where I can switch the language just for that particular card / addon without changing my whole HA system ?

I am currently working on a productive system with a full image backup each night so I just wanna avoid major troubles as long as my wife has gone cause she does not to love to deal with language changes, she needs just one that stays the same regardless if english or german, but not 2. 

Hope that the next will be the release candidate. 

Thx

**Comment by typxxi** [comment]
> Update ok 0625-09
> 
> If ok I'll be able to realease the official version of HACS.

I have had a quick look over the english version and beyond the changes needed mentioned above there are just 2 points beyond those :

#1 "Show state ?" instead of "Show State" 
#2 some are missing the orientation where I had to struggle in the beginning which are avaiable in the german version like "Status zeigen (oben links) ?" which means "Show state (upper left) ?"

In the beginning that helped me a lot.

Final advice in my opinion for the roll out: turn off as default for the "allow account switch ?" cause right now we both know they are without benefit so why should you bring them to the public if this would only raise unneccessary questions right now in my opinion.

**Comment by matteos1** [comment]
There is the possibility to add more user?
